Simulation/Factory/Station/Station.csproj

```diff
@@ -1,17 +1,16 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp1.1</TargetFramework>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
2.0 is out of support in a month or so, let's go with 2.2.

Reply:
its just the api level, no need here to bump it up...